packages/core/src/Stores/ui-store.js

```diff
@@ -95,6 +95,11 @@ export default class UIStore extends BaseStore {
     // UI Focus retention
     @observable current_focus = null;
 
+    // Mobile
+    @observable should_show_toast_error = false;
+    @observable mobile_toast_error = '';
+    @observable mobile_toast_timeout = 1500;
+
```

Comment:
The component using these is in Trader. We can manage the state in Trader itself.